lib/ios/RNNCommandsHandler.m

```diff
@@ -47,6 +49,7 @@ - (instancetype)initWithStore:(RNNStore*)store controllerFactory:(RNNControllerF
 	_stackManager = stackManager;
 	_overlayManager = overlayManager;
 	_mainWindow = mainWindow;
+    _isRootLaunchScreen = YES;
```

Comment:
Can you check if your tab settings are different than in this project? Many of the tabs are in correct 🤔

Reply:
Yeah I don't like tabs either but consistency is key 🔑  :)